mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -248,6 +251,10 @@ pub enum Error {
     InvalidShareProtocol { sp_value: i32 },
     #[snafu(display("Invalid NvmeAnaState value {}", ana_value))]
     InvalidNvmeAnaState { ana_value: i32 },
+    #[snafu(display("Invalid NVMe controller ID range for nexus {}", name))]
+    InvalidNvmeControllerIdRange { name: String },
+    #[snafu(display("Invalid NVMe reservation key for nexus {}", name))]
+    InvalidNvmeReservationKey { name: String },
```

Comment:
Do we really need to keep adding new errors for invalid arguments? Why can't we have that FAR more generic? like InvalidArgument { my details } ?

Reply:
Switched to `InvalidArguments` error type common to both.